tests/pytorch_tests/function_tests/test_tensorboard_writer.py

```diff
@@ -79,16 +128,51 @@ def setUp(self):
         kpi = mct.KPI()
         base_config, _ = get_op_quantization_configs()
         tpc_model = generate_tp_model_with_activation_mp(
-                base_cfg=base_config,
-                mp_bitwidth_candidates_list=[(8, 8), (8, 4), (8, 2),
-                                             (4, 8), (4, 4), (4, 2),
-                                             (2, 8), (2, 4), (2, 2)])
-        tpc = generate_pytorch_tpc(name='mp_pytorch_tpc', tp_model=tpc_model)
+            base_cfg=base_config,
```

Comment:
This file seems very similar to the keras one. Maybe can be unified through a base class?